exporters/metric/datadog/example_test.go

```diff
@@ -104,7 +100,7 @@ func ExampleExporter() {
 	}
 
 	// Output:
-	// myrecorder.max:100|g|#env:dev,l:1,service.name:ExampleExporter,telemetry.sdk.language:go,telemetry.sdk.name:opentelemetry,telemetry.sdk.version:1.2.0
+	// myrecorder:151|c|#env:dev,l:1,service.name:ExampleExporter,telemetry.sdk.language:go,telemetry.sdk.name:opentelemetry,telemetry.sdk.version:1.2.0
```

Comment:
It took a second to figure out why this is the outcome This code ignores all possible Histogram aggregations, now that we've also removed raw `Points` and `MinMaxSumCount`, except for Count. This would be a less confusing example test, therefore, if the instrument above were changed to a Counter and not a Histogram.

Suggest: change to metric.Must(meter).NewInt64Counter("mycounter") above and use `myrcounter:151|c` here.

Reply:
done.